xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-api/src/main/resources/liveDataConfiguration.json

```diff
@@ -47,7 +47,8 @@
       {"id": "html"},
       {"id": "xObjectProperty"},
       {"id": "actions"},
-      {"id": "boolean"}
+      {"id": "boolean"},
+      {"id": "docTitle"}
```

Comment:
``docTitle`` (and ``xObjectProperty``) displayers are not generic (from the point of view of the live data API) so they shouldn't be here. They are specific to a live data source. I'd move this configuration to the livetable module for now if possible (hoping that the merge will work as expected, i.e. keep the standard displayers and add the source-specific ones).